### PR TITLE
Use console.materialize.com instead of cloud.materialize.com

### PR DESCRIPTION
**Pull request by def-**
See discussion in https://materializeinc.slack.com/archives/CM7ATT65S/p1706100390895369

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

**Review by nrainer-materialize — APPROVED** [review]

**Comment by arusahni** [comment]
Sorry, I think there was some lack of precision in the explanation: cloud services are still hosted under cloud.materialize.com. The web console is moved to console.materialize.com.  I believe the URLs in this PR are valid and should remain as-is.

**Comment by def-** [comment]
I updated the PR to only change the actually broken URLs. @arusahni If they are supposed to work, unfortunately they don't.

File: doc/developer/platform/ux.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ either.
 
 ### Global API
 
-The global API is a REST API hosted at <https://cloud.materialize.com/api>. It
+The global API is a REST API hosted at <https://console.materialize.com/api>. It
```
Comment (arusahni) [inline doc/developer/platform/ux.md:212]:
```suggestion
The global API is a REST API hosted at <https://api.cloud.materialize.com/>. It
```

File: misc/python/materialize/cloudtest/util/jwt_key.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def make_jwt(tenant_id: str) -> str:
             "materialize.environment.write",
         ],
         "tenantId": tenant_id,
-        "iss": "https://cloud.materialize.com",
+        "iss": "https://console.materialize.com",
```
Comment (arusahni) [inline misc/python/materialize/cloudtest/util/jwt_key.py:62]:
```suggestion
        "iss": "https://admin.cloud.materialize.com",
```

File: doc/developer/platform/ux.md [hunk 3]
```diff
@@ -1087,7 +1087,7 @@ was Snowflake's approach.
 [dbt]: https://www.getdbt.com
 [frontegg-api]: https://docs.frontegg.com/reference/getting-started-with-your-api
 [frontegg]: https://frontegg.com
-[global-api-spec]: https://cloud.materialize.com/api/schema/browse
+[global-api-spec]: https://console.materialize.com/api/schema/browse
```
Comment (arusahni) [inline doc/developer/platform/ux.md:1090]: I'm not sure where we currently host this (maybe @evanharmon does), but it is not hosted under `console.*`.

**Review by arusahni — CHANGES_REQUESTED** [review]

File: doc/developer/platform/ux.md (hunk 3 above)
Comment (evanharmon) [reply doc/developer/platform/ux.md]: The updated change to `<https://api.cloud.materialize.com>` is correct.

Comment (def-) [reply doc/developer/platform/ux.md]: But there is now schema/browse there: https://api.cloud.materialize.com/schema/browse

Comment (evanharmon) [reply doc/developer/platform/ux.md]: I can confirm the schema isn't publicly available for the global-api.

Comment (def-) [reply doc/developer/platform/ux.md]: So we should just remove this link entirely from the ux page?

Comment (arusahni) [reply doc/developer/platform/ux.md]: I think that's for the best.

**Pull request merged**